docs/python/inference/examples_md.rst

```diff
@@ -4,21 +4,20 @@
 
     Gallery of examples
     ===================
-    
+
     This series of examples briefly goes into the main
-    feature *ONNX Runtime* implements. Each of them run in a 
+    feature *ONNX Runtime* implements. Each of them run in a
     few seconds and relies on machine learned models
     trained with `scikit-learn <http://scikit-learn.org/stable/>`_.
-    
+
     .. toctree::
         :maxdepth: 1
         :caption: Contents:
-        
+
         auto_examples/plot_load_and_predict
         auto_examples/plot_common_errors
         auto_examples/plot_train_convert_predict
         auto_examples/plot_pipeline
         auto_examples/plot_backend
-        auto_examples/plot_convert_pipeline_vectorizer
```

Comment:
why is this part of this PR?

Reply:
The example was flagged as inappropriate.

Reply:
I've updated the PR description